src/clr-angular/emphasis/alert/alert.ts

```diff
@@ -3,28 +3,52 @@
  * This software is released under MIT license.
  * The full license information can be found in LICENSE in the root directory of this project.
  */
-import { ChangeDetectorRef, Component, EventEmitter, Input, Optional, Output } from '@angular/core';
+import {
+  ChangeDetectorRef,
+  Inject,
+  ElementRef,
+  Component,
+  EventEmitter,
+  Input,
+  Optional,
+  Output,
+  AfterViewInit,
+  PLATFORM_ID,
+  ContentChildren,
+  QueryList,
+} from '@angular/core';
+import { isPlatformBrowser } from '@angular/common';
 
 // providers
 import { AlertIconAndTypesService } from './providers/icon-and-types.service';
 import { MultiAlertService } from './providers/multi-alert.service';
 import { isBooleanAttributeSet } from '../../utils/component/is-boolean-attribute-set';
 import { ClrCommonStringsService } from '../../utils/i18n/common-strings.service';
+import { AriaLiveService, AriaLivePoliteness } from '../../utils/a11y/aria-live.service';
+import { ClrAlertText } from './alert-text';
 
 @Component({
   selector: 'clr-alert',
-  providers: [AlertIconAndTypesService],
+  providers: [AlertIconAndTypesService, AriaLiveService],
   templateUrl: './alert.html',
   styles: [':host { display: block; }'],
 })
-export class ClrAlert {
+export class ClrAlert implements AfterViewInit {
   constructor(
     public iconService: AlertIconAndTypesService,
     public cdr: ChangeDetectorRef,
     @Optional() public multiAlertService: MultiAlertService,
-    public commonStrings: ClrCommonStringsService
+    public commonStrings: ClrCommonStringsService,
+    private ariaLiveService: AriaLiveService,
+    @Inject(PLATFORM_ID) private platformId: Object,
```

Comment:
Do we need this platformId anymore? There could be things we longer need in the imported modules too.